### PR TITLE
test: Cypress - Flaky test fix

### PR DESCRIPTION
**Pull request by Vijetha-Kaja**
## Description

**Fixed below flaky tests**

- Fork_Template_spec.js
- JSOnLoad2_Spec.ts
- Omnibar_spec.js
- RangeSlider_spec.ts

## Type of change

- Flaky test fix

## How Has This Been Tested?
- Cypress test runs

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/cypress/integration/Regression_TestSuite/ServerSideTests/OnLoadTests/JSOnLoad2_Spec.ts [hunk 1]
```diff
@@ -158,12 +158,12 @@ describe("JSObjects OnLoad Actions tests", function () {
     _.deployMode.DeployApp();
     for (let dialog = 1; dialog <= 5; dialog++) {
       _.agHelper.ClickButton("Yes");
-      _.agHelper.Sleep(2000);
+      _.agHelper.Sleep(500);
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ServerSideTests/OnLoadTests/JSOnLoad2_Spec.ts:161]: Is this enough & passing - How many times did we test in CI? I scripted this & seen less time causes it to be flaky.

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/OtherUIFeatures/Omnibar_spec.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 const omnibar = require("../../../../locators/Omnibar.json");
 const dsl = require("../../../../fixtures/omnibarDsl.json");
 const commonlocators = require("../../../../locators/commonlocators.json");
-import { ObjectsRegistry } from "../../../../support/Objects/Registry";
-
-const ee = ObjectsRegistry.EntityExplorer;
+import {
+  agHelper,
+  entityExplorer,
+} from "../../../../support/Objects/ObjectsCore";
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/OtherUIFeatures/Omnibar_spec.js:7]: When we have below line, why is this import needed?

Comment (Vijetha-Kaja) [reply app/client/cypress/integration/Regression_TestSuite/ClientSideTests/OtherUIFeatures/Omnibar_spec.js:7]: i think it got added automatically when i have modified code. Will remove it

**Review by Aishwarya-U-R — APPROVED** [review]
